src/docs/development/add-to-app/android/add-flutter-screen.md

```diff
@@ -14,104 +14,29 @@ see-through, translucent screen. Both options are described in this guide.
-
-To use `FlutterApplication` directly, simply declare it as the `Application`
-class in the `AndroidManifest.xml` file:
-
-```xml
-<application
-  android:name="io.flutter.app.FlutterApplication"
-  // other properties omitted
-  >
-  <!-- content omitted -->
-</application>
-```
-
-Now, as soon as your application is created, Flutter initializes.
-
-#### Option 2: Extend FlutterApplication
-
-Extending `FlutterApplication` automatically initializes Flutter when
-Android creates your application. First, create an `Application` class that
-extends `FlutterApplication`:
-
-```java
-public class MyApplication extends FlutterApplication {
-  // no implementation is necessary.
-}
-```
-
-Second, register your custom `Application` subclass in the
-`AndroidManifest.xml` file:
-
-```xml
-<application
-  android:name=".MyApplication"
-  // other properties omitted
-  >
-  <!-- content omitted -->
-</application>
-```
-
-Replace `MyApplication` with your `Application` subclass name.
-
-#### Option 3: Manually initialize Flutter
-
-If you'd like to control precisely where and when Flutter is initialized, you
-can initialize Flutter manually. This must occur before you attempt to display
-any Flutter content, or execute any Dart code. Initialization only needs to
-happen once within the application process - it does not need to be repeated
-for every Flutter experience.
-
-To manually initialize Flutter, execute the following code at the desired
-location:
-
-```java
-FlutterMain.startInitialization(appContext);
-FlutterMain.ensureInitializationComplete(appContext, null);
-```
-
-The call to `ensureInitializationComplete()` is a blocking call. Make sure you
-invoke it at a place and time that can afford a brief processing pause.
-
-### Step 3: Add FlutterActivity to AndroidManifest.xml
+### Step 1: Add FlutterActivity to AndroidManifest.xml
 
 Flutter provides `FlutterActivity` to display a Flutter experience within an
-Android `Activity`. Like any other `Activity`, `FlutterActivity` must be
+Android app. Like any other `Activity`, `FlutterActivity` must be
 registered in your `AndroidManifest.xml`. Add the following XML to your
 `AndroidManifestxml` file under your `application` tag:
 
 ```xml
 <activity
-  android:name=".YourFlutterActivityName"
-  android:launchMode="singleTop"
+  android:name="io.flutter.embedding.android.FlutterActivity"
   android:theme="@style/LaunchTheme"
   android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
   android:hardwareAccelerated="true"
   android:windowSoftInputMode="adjustResize"
   />
 ```
 
-Replace `YourFlutterActivityName` with the name of your Flutter `Activity`.
+The reference to `@style/LaunchTheme` can be replaced by any Android theme that
+you'd like to apply to your `FlutterActivity`. The choice of theme dictates the
+colors of system chrome and the background color of the `FlutterActivity` just
```

Comment:
"colors of system chrome" feels like an odd phrase and I'm not sure I follow what you mean.

Reply:
I tried another approach. Let me know if it's still unclear.